src/molecule_plugins/gce/driver.py

```diff
@@ -94,10 +94,10 @@ def login_cmd_template(self):
         connection_options = " ".join(self.ssh_connection_options)
 
         return (
-            "ssh {{address}} "
-            "-l {{user}} "
-            "-p {{port}} "
-            "-i {{identity_file}} "
+            "ssh {address} "
+            "-l {user} "
+            "-p {port}} "
```

Comment:
Looks like there's an extra closing curly brace for line 99, `-p {port}}`, which should read `-p {port}`

Reply:
Good catch - Fixed that up.